staging/src/k8s.io/component-base/logs/logs.go

```diff
@@ -176,6 +186,10 @@ func InitLogs() {
 		// Otherwise LoggingConfiguration.Apply will do this.
 		klog.StartFlushDaemon(logFlushFreq)
 	}
+
+	// This is the default in Kubernetes. Options.ValidateAndApply
+	// will override this with the result of a feature gate check.
+	klog.EnableContextualLogging(false)
```

Comment:
my understanding is that this is the minimal change *required* for 1.24 to ensure the feature is not used by default in 1.24, right?

Reply:
See reply in https://github.com/kubernetes/kubernetes/pull/108995#issuecomment-1081433903